packages/utils/src/json-schemas/index.ts

```diff
@@ -35,15 +37,23 @@ export function validateSnapJsonFile(fileName: NpmSnapFileNames, content: unknow
   }
 
   if (errors && errors.length !== 0) {
-    throw new Error(`${
-      errors.reduce((allErrors, errorObject: { instancePath?: string, message?: string } = {}) => {
-        const { instancePath, message = 'unknown error' } = errorObject;
-        const currentString = instancePath
-          ? `\t${instancePath}\n\t${message}\n\n`
-          : `\t${message}\n\n`
+    throw new Error(
```

Comment:
What's going on here? 😅

Reply:
This file wasn't formatted before (ignored by ESLint), so I just formatted it.